src/refract/JSONSchemaVisitor.cc

```diff
@@ -269,18 +282,18 @@ namespace refract
                         // want them in the schema, otherwise skip
                         // empty ones
                         if (allEmpty) {
-                            JSONSchemaVisitor renderer;
+                            JSONSchemaVisitor v;
```

Comment:
Variable shadowing takes placd hence the renaming and becase the scope is 3 lines i wantdd something short